generateBins.py

```diff
@@ -22,6 +30,48 @@
 # Following bins are for Rwanda
 POPULATION_BINS = [2125, 5640, 8989, 12108, 15577, 20289, 27629, 49378, 95262, 286928]
 
+# getting data ready for binning
+# data should be 1-dimensional array, python list or iterable
+myArray  = np.loadtxt("GIS\\rwa_population.asc", skiprows = 6)
```

Comment:
Remember that the GIS file can change, so avoid hardcoding filenames for production code. This load also shouldn't take place until the function where the data needs to be loaded.